_posts/2023-06-27-release.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Release 2023.26
+author: Priya
+date: 2023-06-27
+category: Jekyll
+layout: post
+---
+
+### Enhancements
+
+### Improved Date difference rule
+
+Enhanced the Date difference rule to ensure that the result of the date difference is always a positive integer when the start date is earlier than the end date.
+
+
+### Defect fixes
+
+* Resolved the issue where the "Move to" UI in the Folder feature was broken.
+* Fixed the issue where not all assets of an account were transferred to the target user.
+* Resolved the issue of invalid HTML presence in the DOM, causing the element to overflow and be absent from the viewport.
+* Fixed the issue where downloading as CSV would result in downloading the CSV file twice.
+* Resolved the issue where resetting the foreign view in the Join rule did not update the dependent data sources.
+* Fixed the issue where Branch out did not exit the error state when columns were reordered and all columns were correctly mapped.
+* Resolved the issue where the Pipeline remained in an error state even after fixing the rule.
+* Fixed the issue where creating a data source with combine mode and a USC column failed in Snowflake.
```

Comment:
Fixed the issue where creating a data source in combine mode and a unique sequence column failed for Snowflake integration

Reply:
fixed